projects/addon-editor/components/edit-link/edit-link.component.ts

```diff
@@ -48,21 +53,27 @@ export class TuiEditLinkComponent {
         @Inject(TUI_EDITOR_LINK_TEXTS)
         readonly texts$: TuiInjectionTokenType<typeof TUI_EDITOR_LINK_TEXTS>,
         @Inject(TuiTiptapEditorService) private readonly editor: AbstractTuiEditor,
+        @Inject(TUI_EDITOR_OPTIONS)
+        private readonly options: TuiEditorOptions,
     ) {}
 
+    get defaultProtocol(): TuiEditorLinkProtocol {
+        return this.options?.linkOptions?.()?.protocol ?? TUI_EDITOR_LINK_HTTP_PREFIX;
```

Comment:
`this.options` cannot be falsy. And shouldn't default fallback be https?

Reply:
fixed